client/src/pages/GrantAccessForm.js

```diff
@@ -21,17 +21,21 @@ const GrantAccessForm = () => {
 	});
 
 	const [resgisterStatus, setRegisterStatus] = useState("");
+	const [validationErrors, setValidationErrors] = useState([]);
 
 	async function postDetails(endpoint, data) {
 		try {
-			const res = await post(`${endpoint}`, data);
-			if (res) {
+			const res = await post(endpoint, data);
+			console.log(res);
+			if (res || res.code !== "VALIDATION_ERROR") {
```

Comment:
When you get a 403, will this code be executed or will the code on the catch clause be executed?

Reply:
The catch block will be excuted. But I can add error handling for error 403.

Reply:
If the catch block is executed, then `res.code` will never be `VALIDATION_ERROR` here, so this condition is unnecessary.

And yes, you have to check the error code in the catch block, because if the error is a 500, then `errors` will not be set.